app/Place.php

```diff
@@ -58,6 +61,11 @@ public function getPageTitle(): string
         return "{$this->name}{$midName}".config('app.name', '');
     }
 
+    #SCOPES
+    public function scopeOpened($query)
+    {
+        return $query->where('is_closed', 0);
+
```

Comment:
Just as feedback, but you where missing a closing semi colon here!

Reply:
Resolving conflict within Github gone wrong :s 
Thanks for fixing it!